packages/mdc-tab-scroller/mdc-tab-scroller.scss

```diff
@@ -0,0 +1,50 @@
+/**
+  * @license
+  * Copyright 2018 Google Inc. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License")
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+*/
+
+@import "@material/animation/variables";
+
+.mdc-tab-scroller {
+  overflow-y: hidden;
+}
+
+.mdc-tab-scroller__scroll-area {
+  -webkit-overflow-scrolling: touch;
+  display: flex;
+  margin-bottom: -18px;
+  overflow-x: scroll;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+}
+
+.mdc-tab-scroller__scroll-area--animating {
```

Comment:
nit...maybe...but should it be `mdc-tab-scroller--animating`? I think maybe BEM is easier to understand when the modifiers are only on the block element.

Reply:
Sounds good.